CONTRIBUTING.md

```diff
@@ -22,6 +22,4 @@ Inspected 36/63 (chai)
 {"message":"Server Error: Sorry, this diff is taking too long to generate.","errors":[{"resource":"Comparison","field":"diff","code":"not_available"}],"documentation_url":"https://developer.github.com/v3/repos/commits/#compare-two-commits"}
 ```
 
-As your token provides access to your account and thus could be abused if made public.
-
-If you did provide your token, you should [revoke it](https://github.com/settings/tokens) immediately.
+Your token provides access to your account and thus could be abused if made public. If you did provide your token, you should [revoke it](https://github.com/settings/tokens) immediately. We do not accept responsibility for any damage caused by you piblishing your access token in this repo.
```

Comment:
typo: "piblishing" --> "publishing"

Reply:
Also maybe I'd suggest adding hard line breaks in the [appropriate places](http://rhodesmill.org/brandon/2012/one-sentence-per-line/). Single line breaks allow the rendered text to continue flowing as a single paragraph, but the diffs are easier to read.